commands/verify.js

```diff
@@ -96,25 +55,32 @@ async function getDetails(message,username) {
       let realName = author_detail[i][1];
 
 			let part = realName.split(" ")
+			let q=0, lastName="abc";
```

Comment:
Better `lastName = part[ part.length - 1 ]`. Ain't it?

Reply:
yes